File: src/main/java/com/tristanbomb/powerofpaint/proxy/ClientProxy.java
package com.tristanbomb.powerofpaint.proxy;

import com.tristanbomb.powerofpaint.block.ModBlocks;
import com.tristanbomb.powerofpaint.item.ModItems;

public class ClientProxy extends CommonProxy {
	public void preInit() {
		super.preInit();
		ModBlocks.initModels();
		ModItems.initModels();
	}
	public void init() {
		
	}	
	public void postInit() {
		
	}
}

File: src/main/kotlin/tristanbomb/powerofpaint/proxy/ClientProxy.kt
package tristanbomb.powerofpaint.proxy

import net.minecraftforge.client.event.ModelRegistryEvent
import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.common.event.FMLInitializationEvent
import net.minecraftforge.fml.common.event.FMLPostInitializationEvent
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
import net.minecraftforge.fml.relauncher.Side
import tristanbomb.powerofpaint.PowerOfPaint
import tristanbomb.powerofpaint.ModObjects

@Mod.EventBusSubscriber(Side.CLIENT)
class ClientProxy : CommonProxy() {
    override fun preInit(e: FMLPreInitializationEvent) {
        super.preInit(e)
    }

    override fun init(e: FMLInitializationEvent) {
        super.init(e)

    }
    override fun postInit(e: FMLPostInitializationEvent) {
        super.postInit(e)
    }

    @SubscribeEvent
    open fun registerModels(e: ModelRegistryEvent) {
        for (i in ModObjects.blocks) {
            i.initModel()
        }
        for (i in ModObjects.items) {
            i.initModel()
        }
    }
}

